app/src/main/kotlin/com/mapbox/vision/teaser/SafetyFragment.kt

```diff
@@ -0,0 +1,238 @@
+package com.mapbox.vision.teaser
+
+import android.animation.Animator
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.animation.AccelerateDecelerateInterpolator
+import androidx.fragment.app.Fragment
+import com.mapbox.services.android.navigation.v5.navigation.NavigationConstants
+import com.mapbox.services.android.navigation.v5.utils.DistanceFormatter
+import com.mapbox.services.android.navigation.v5.utils.LocaleUtils
+import com.mapbox.vision.mobile.core.models.Country
+import com.mapbox.vision.mobile.core.models.detection.DetectionClass
+import com.mapbox.vision.safety.VisionSafetyManager
+import com.mapbox.vision.safety.core.VisionSafetyListener
+import com.mapbox.vision.safety.core.models.CollisionDangerLevel
+import com.mapbox.vision.safety.core.models.CollisionObject
+import com.mapbox.vision.safety.core.models.RoadRestrictions
+import com.mapbox.vision.teaser.models.UiSign
+import com.mapbox.vision.teaser.utils.SoundsPlayer
+import com.mapbox.vision.teaser.utils.classification.SignResources
+import com.mapbox.vision.teaser.utils.requireMainActivity
+import com.mapbox.vision.teaser.utils.requireVisionManager
+import com.mapbox.vision.teaser.utils.runOnUiThread
+import com.mapbox.vision.teaser.view.hide
+import com.mapbox.vision.teaser.view.show
+import kotlinx.android.synthetic.main.fragment_safety.*
+
+class SafetyFragment : Fragment() {
+
+    companion object {
+        val TAG: String = SafetyFragment::class.java.simpleName
+        private const val CALIBRATION_READY_VALUE = 1f
+        fun newInstance() = SafetyFragment()
```

Comment:
IMO it's not needed since it just calls noarg constructor, caller will look simpler with plain `= SafetyFragment()`

Reply:
Maybe keep newInstance() for common style for Fragment creation?
It is common practice and used in ReplayFragment and RecordFragment to save data to bundle

Reply:
Sounds good, lets keep it